app/models/maintenance_tasks/task.rb

```diff
@@ -70,7 +70,7 @@ def available_tasks
       # An input to upload a CSV will be added in the form to start a Run. The
       # collection and count method are implemented.
       def csv_collection(in_batches: nil)
-        unless defined?(ActiveStorage)
+        unless MaintenanceTasks.active_storage_installed?
```

Comment:
We should not use the new method here. This runs at boot time when the tasks are loaded.
```console
SECRET_KEY_BASE="$(bin/rails secret)" bin/rails server -e production
```

This works on main but not on the branch.